src/init.cpp

```diff
@@ -1545,6 +1545,17 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
     }
 #endif // ENABLE_MINING
 
+    // Start the thread that notifies listeners of transactions that have been
+    // recently added to the mempool, or have been added to or removed from the
+    // chain. We perform this before step 10 (import blocks) so that the
```

Comment:
It might be useful to link to this URL:

https://en.bitcoin.it/wiki/Bitcoin_Core_0.11_(ch_3):_Initialization_and_Startup

since the comment refers to "step 10".

Reply:
Ooops...  I see "Step 10" in `init.cpp` nevermind.